File: 16_subplot.m
% Generate some data
x = linspace(0, 10, 100);
y1 = sin(x);
y2 = cos(x);

% Create subplots
figure;

subplot(2, 1, 1)  % (rows, columns, plot_number)
plot(x, y1)
title('Sin Wave')

subplot(2, 1, 2)
plot(x, y2)
title('Cosine Wave')

File: 16_subplot.py
import numpy as np
import matplotlib.pyplot as plt

# Generate some data
x = np.linspace(0, 10, 100)
y1 = np.sin(x)
y2 = np.cos(x)

# Create subplots
plt.figure(figsize=(10, 5))

plt.subplot(2, 1, 1)  # (rows, columns, plot_number)
plt.plot(x, y1)
plt.title('Sin Wave')

plt.subplot(2, 1, 2)
plt.plot(x, y2)
plt.title('Cosine Wave')

plt.tight_layout()  # Adjust layout to prevent overlap
plt.show()

